static/app/components/events/searchBar.tsx

```diff
@@ -203,7 +205,9 @@ function SearchBar(props: SearchBarProps) {
       kind: FieldKind.FIELD,
     };
 
-    return omit(combinedTags, omitTags ?? []);
+    const list = omit(combinedTags, omitTags ?? []);
+    transaction.finish();
```

Comment:
If you want a count of the tags, you should do tag grouping so you can use tag heat map to nicely lay out if perf is related to `n` tags.  See https://github.com/getsentry/sentry/blob/93ebbc15040bcd7a162eaac0dd95d9bce73b275b/static/app/views/performance/transactionSummary/transactionSpans/spanDetails/content.tsx#L103-L106

Also, you can make a child span with just timings _after_ the measurement if it's tight enough you are worried about the sdk code causing overhead.

```ts
      transaction.startChild({
        description: ...
        op: ...
        startTimestamp: start,
        endTimestamp: end,
      });
      transaction.setTag(....) // outside of the timing.
    ```